backend/src/main/kotlin/blue/mild/covid/vaxx/error/ExceptionHandling.kt

```diff
@@ -45,6 +40,11 @@ fun Application.installExceptionHandling() {
             logger.warn { cause.message }
             call.errorResponse(HttpStatusCode.BadRequest, "Bad request. ${cause.message}")
         }
+
+        exception<Exception> { cause ->
+            logger.error(cause) { "Exception occurred in the application: ${cause.message}" }
+            call.errorResponse(HttpStatusCode.InternalServerError, cause.message)
```

Comment:
A ven bych nevyhazoval `cause.message` protoze to je seccurity issue, ale "Internal server error, please contact support" a treba i vygeneroval nejaky GUID, ktery bych ulozit k te exception do logu i poslal uzivateli, tj.  "Internal server error, please contact support with issue id {GUID}."

Reply:
jo, done